src/GraphQL.SystemTextJson/StringExtensions.cs

```diff
@@ -36,5 +40,81 @@ public static Inputs ToInputs(this string json)
         /// <returns>Dictionary.</returns>
         public static Dictionary<string, object> ToDictionary(this string json)
             => JsonSerializer.Deserialize<Dictionary<string, object>>(json, _jsonOptions);
+
+        /// <summary>
+        /// Deserializes a JSON-formatted string of data into the specified type.
+        /// Any <see cref="Dictionary{TKey, TValue}">Dictionary&lt;string, object&gt;</see> objects will be deserialized
+        /// into the proper format for passing to <see cref="InputsExtensions.ToInputs(Dictionary{string, object})"/>.
+        /// </summary>
+        public static T FromJson<T>(this string json)
+            => JsonSerializer.Deserialize<T>(json, _jsonOptions);
+
+        /// <summary>
+        /// Deserializes a JSON-formatted stream of data into the specified type.
+        /// Any <see cref="Dictionary{TKey, TValue}">Dictionary&lt;string, object&gt;</see> objects will be deserialized
+        /// into the proper format for passing to <see cref="InputsExtensions.ToInputs(Dictionary{string, object})"/>.
+        /// </summary>
+        public static ValueTask<T> FromJsonAsync<T>(this System.IO.Stream stream, CancellationToken cancellationToken = default)
+            => JsonSerializer.DeserializeAsync<T>(stream, _jsonOptions, cancellationToken);
+
+        /// <summary>
+        /// Converts a JSON element into an <see cref="Inputs"/> object.
+        /// </summary>
+        public static Inputs ToInputs(this JsonElement obj)
+        {
+            if (obj.ValueKind == JsonValueKind.Null)
```

Comment:
```suggestion
            if (obj == null)
```

Reply:
JsonElement is a struct.  It can't be null.

Reply:
Right now if `default` was passed in to the method, it would execute this code:
```cs
            if (obj.ValueKind != JsonValueKind.Object)
                throw new InvalidOperationException("This element is not an object element");
```

Reply:
But it does check for a `null` element and the test show this works correctly.

Reply:
Note that for Newtonsoft.Json, the object is a class and would actually be `null`.  This is properly tested for in the tests I added.

Reply:
> JsonElement is a struct. It can't be null.

🤦‍♂️